stories/blocks.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import Blocks from '../source/features/shared/blocks/components/Blocks';
+import { ThemeDecorator } from './support/ThemeDecorator';
+
+const blocks = [
+  {
+    block: 20051,
+    createdAt: 1568366883000,
+    createdBy: 'af2800c',
+    epoch: 138,
+    output: 11189.647356,
+    size: 1024,
+    slots: 21600,
+    transactions: 1,
+  },
+  {
+    block: 20051,
+    createdAt: 1568366883000,
+    createdBy: '6c9e149',
+    epoch: 139,
+    output: 11189.647356,
+    size: 1024,
+    slots: 21600,
+    transactions: 2,
+  },
+  {
+    block: 20051,
+    createdAt: 1568366883000,
+    createdBy: 'af2800c',
+    epoch: 137,
+    output: 11189.647356,
+    size: 1024,
+    slots: 21600,
+    transactions: 3,
+  },
+  {
+    block: 20051,
+    createdAt: 1568366883000,
+    createdBy: '6c9e149',
+    epoch: 139,
+    output: 11189.647356,
+    size: 1024,
+    slots: 21600,
+    transactions: 4,
+  },
+  {
+    block: 20051,
+    createdAt: 1568366883000,
+    createdBy: 'e1496c9',
+    epoch: 140,
+    output: 11189.647356,
+    size: 1024,
+    slots: 21600,
+    transactions: 5,
+  },
+];
+
+storiesOf('Blocks', module)
+  .addDecorator(story => <ThemeDecorator>{story()}</ThemeDecorator>)
+  .add('Blocks', () => (
+    <div
+      style={{
+        backgroundColor: '#121326',
```

Comment:
@DeeJayElly could you change this approach to CSS styling? This way you can use SCSS variables, that will be used for the different network versions. For instance, `#121326` would be `$primary-bg`.

Reply:
Fixed